src/acpi/src/arch/x86_64.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::mem::size_of;
+
+use vm_memory::ByteValued;
+
+use crate::fadt::Fadt;
+
+// Disable VGA probing
+const IAPC_BOOT_ARG_FLAGS_VGA_NOT_PRESENT: u8 = 2;
+// Do not enable MSI
+const IAPC_BOOT_ARG_FLAGS_MSI_NOT_PRESENT: u8 = 3;
+// Do not enable ASPM control
+const IAPC_BOOT_ARG_FLAGS_PCI_ASPM: u8 = 4;
```

Comment:
Would it not be easier to define them as their masks?
```suggestion
// Disable VGA probing
const IAPC_BOOT_ARG_FLAGS_VGA_NOT_PRESENT: u8 = 1 << 2;
// Do not enable MSI
const IAPC_BOOT_ARG_FLAGS_MSI_NOT_PRESENT: u8 = 1 << 3;
// Do not enable ASPM control
const IAPC_BOOT_ARG_FLAGS_PCI_ASPM: u8 = 1 << 4;
```